### ISSUE TITLE
Add an error dialog for desktop panics due to unsupported/lack of GPU

### ISSUE THREAD

**Issue by sombraguerrero** [ruffle-rs/ruffle#13312]
We have a sufficient number of regular complaints about the desktop app crashing due to lack of hardware acceleration support that we should expose that error in a message dialog in layman's terms so it is understood by the masses.

**Comment by n0samu** [comment]
Related: we should change the hardcoded "Ruffle does not support OpenGL on Windows" error message to reflect the user's actual platform and requested backend.
